cli/azd/pkg/graphsdk/application_request_builders_test.go

```diff
@@ -0,0 +1,196 @@
+package graphsdk_test
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/azure/azure-dev/cli/azd/pkg/convert"
+	"github.com/azure/azure-dev/cli/azd/pkg/graphsdk"
+	"github.com/azure/azure-dev/cli/azd/test/mocks"
+	graphsdk_mocks "github.com/azure/azure-dev/cli/azd/test/mocks/graphsdk"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetApplicationList(t *testing.T) {
+	t.Run("Success", func(t *testing.T) {
+		expected := []graphsdk.Application{
+			{
+				Id:          convert.RefOf("1"),
```

Comment:
the azure sdk for go comes with a package `to` which provides `to.Ptr()`. It do the same as `convert.RefOf()`.

I think we should use the one from the sdk, instead of maintaining one in our code

Reply:
Maybe - but going to hold off on this one.